src/options.rs

```diff
@@ -128,6 +128,11 @@ pub struct BuildOptions {
     /// optimized targets. This flag allows you to opt out of it.
     #[structopt(long)]
     pub strip_dead_code: bool,
+
+    /// By default a 'fuzzing' compilation configuration is set. This flag
+    /// allows you to opt out of it.
```

Comment:
I think users will be able to better recognize what we are talking about here if we have the `cfg(..)` wrapper, like so:

```suggestion
    /// By default, the `cfg(fuzzing)` compilation configuration is set. This flag
    /// allows you to opt out of it.
```

Reply:
Applied :)